src/queries/project/index.ts

```
@@ -0,0 +1,175 @@
+import {
+  CreateProjectRequest,
+  CreateProjectResponse,
+  GetProjectDetailResponse,
+  GetProjectListQuery,
+  GetProjectListResponse,
+  GetProjectTop5Response,
+  UpdateProjectRequest,
+  UpdateProjectResponse,
+} from '@/types/api/Project.types'
+import {
+  GetProjectListQuery,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'GetProjectListQuery'. Consider to delete it or rename it.

'GetProjectListQuery' is defined here:


(lint/suspicious/noRedeclare)